### ISSUE TITLE
ERROR in Sentry CLI Plugin: Cannot read property 'execute' of undefined

### ISSUE THREAD

**Issue by MmtBkn** [getsentry/sentry-webpack-plugin#5]
Hello,

I'm trying to run webpack using following plugin configuration and .sentryclirc file;

```
    plugins: [
        new SentryCliPlugin({
            release: function (hash) {
                return hash.slice(0, 5)
            },
            include: '.',
            ignore: ['node_modules', 'webpack.config.js'],
            configFile: 'sentry.properties'
        })
    ],
```

```
[defaults]
url=https://sentry.io/
org=xxx-llc
project=react

[auth]
token=asdfdsa9a3asdfadsfasdfasdfsdafdasdsfasadff17c721asdfdsafasdfdsaf
```

And I'm getting this error;
`ERROR in Sentry CLI Plugin: Cannot read property 'execute' of undefined`

**Comment by jan-auer** [comment]
This has already been fixed in getsentry/sentry-cli#183 and will be released soon.

**Comment by jan-auer** [comment]
Just released [1.26.1](https://github.com/getsentry/sentry-cli/releases/tag/1.26.1), it should be available within the next 15-30 minutes.
